<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customer Booking</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{

background-image:url(ui/res/bg.gif);

}</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>361</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Customer Info</string>
    </property>
    <widget class="QLineEdit" name="fullname_le">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Full Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Phone Number</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="phonenumber_le">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Check-In</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Check-Out</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="address_le">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
    <widget class="QDateEdit" name="checkin_date_de">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2022</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTimeEdit" name="checkin_time_te">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>12</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
    </widget>
    <widget class="QDateEdit" name="checkout_date_de">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2022</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTimeEdit" name="checkout_time_te">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>170</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>12</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>331</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Room Type</string>
    </property>
    <widget class="QSpinBox" name="singlebed_sb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Single Bed</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="doublebed_sb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Bed Type</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="singlebed_tb">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>950/night</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Double Bed</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="doublebed_tb">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>1200/night</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Family Room</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="family_sb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextBrowser" name="family_tb">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>3600/night</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Amenities</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="petroom_cb">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>210</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pet Room(250/day)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="spa_cb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Spa(500/day)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="breakfast_cb">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Breakfast(250/day)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Reciept</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Amount&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="totalamount_tb">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="placeholderText">
      <string>00000</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="checkout_btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>590</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#checkout_btn{
background-color:qconicalgradient(cx:0.915, cy:0.909091, angle:359.6, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
border: 1px solid gold;
border-radius: 20px 20px;
}</string>
    </property>
    <property name="text">
     <string>Checkout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_btn">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>590</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#exit_btn{
background-color:qconicalgradient(cx:0.915, cy:0.909091, angle:359.6, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
border: 1px solid gold;
border-radius: 20px 20px;
}

</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#EEBC1D;&quot;&gt;BOOKING&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>610</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_lbl">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>620</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>[status]</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Guest Count</string>
    </property>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Adults</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="adults_sb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QSpinBox" name="children_sb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Children</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>490</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Remarks</string>
    </property>
    <widget class="QLineEdit" name="remarks_le">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>841</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(241, 229, 172,70%);
border: 1px solid gold;
border-radius: 20px 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>481</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;ui/res/logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>107</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777212</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0)</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>label_18</zorder>
   <zorder>label_17</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>checkout_btn</zorder>
   <zorder>exit_btn</zorder>
   <zorder>label_13</zorder>
   <zorder>label_14</zorder>
   <zorder>status_lbl</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_6</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="res/Logo.qrc"/>
 </resources>
 <connections/>
</ui>
